package.json

```diff
@@ -17,17 +17,20 @@
     "prototyping"
   ],
   "scripts": {
-    "start": "npm-run-all --continue-on-error --parallel gulp dev:pl",
-    "dev:pl": "webpack-dev-server --config ./apps/pl/webpack.pl.dev.js --hot --progress",
-    "build:pl": "webpack --config ./apps/pl/webpack.pl.prod.js && gulp compile",
+    "start": "npm run dev:pl",
+    "dev:pl": "npm-run-all --continue-on-error --parallel dev:pl:gulp dev:pl:webpack",
+    "dev:pl:webpack": "webpack-dev-server --config ./apps/pl/webpack.pl.dev.js --hot --progress",
+    "dev:pl:gulp": "NODE_ENV='development' gulp default",
+    "build:pl": "NODE_ENV='production' gulp compile; webpack --config ./apps/pl/webpack.pl.prod.js",
+    "dev:drupal": "npm-run-all --continue-on-error --parallel dev:drupal:gulp dev:drupal:webpack",
+    "dev:drupal:webpack": "webpack --watch --progress --config ./apps/drupal/webpack.drupal.dev.js",
+    "dev:drupal:gulp": "NODE_ENV='development' echo Placeholder for Drupal gulp tasks",
```

Comment:
am i correct assuming this placeholder is what will ultimately run the cache clear on rebuild?

Reply:
I'm looking to get some better notes into the readme, but this could be *any* drupal-specific gulp tasks. I actually have a completely non-gulp solution for drupal cache clears here now:

https://github.com/phase2/particle/pull/249/files#diff-3b4f688e110edb25e012b075409b31c4R24

Basically, "run a command after webpack --watch builds each time".